plcc/src/io/FileParser.java

```diff
@@ -389,17 +391,12 @@ protected static boolean isAminoacid(String AAName, Boolean includeNonStandard)
     
     
     /**
-     * Returns true if RNAName is standard RNA name (1-letter code).
-     * @param RNAName RNA name, 1-letter code, capitalized
+     * Returns true if chainIdentity equals polyribonucleotide.
+     * @param chainID Name of the chain i.e. A
      * @return 
      */
-    private static boolean isRNA(String RNAName) {
-        String[] standardAANames = {"A", "G", "C", "G"};
-        if (Arrays.asList(standardAANames).contains(RNAName)) {
-            return true;
-        } else {
-            return false;
-        }
+    protected static boolean isRNAchain(String chainID) {
+        return (CifParser.chainIdentity.get(chainID).equals("polyribonucleotide") ? true : false);
```

Comment:
Returning true if something is true and false when something is false? Why not just return what you get back from the 'equals'?